inc/functions.php

```diff
@@ -187,3 +187,22 @@ function get_cookie_policy_url() : string {
 function get_category_label( string $category ) : string {
 	return consent_category_labels()[ $category ] ?? '';
 }
+
+/**
+ * Retrieve the always allowed categories.
+ *
+ * @return array Returns always allowed categories.
+ */
+function get_always_allowed_categories() {
+	$allowed = apply_filters( 'altis.consent.always_allow_categories', [] );
+	if ( ! empty( $allowed ) ) {
+		return $allowed;
+	}
+
+	$default = [
+		'functional',
+		'statistics-anonymous',
+	];
+
+	return $default;
+}
```

Comment:
What's the reasoning behind this approach? Typically a filter will pass the default values through which can then be edited, but this appears to check if someone has used the filter to modify the array and then if they haven't we return these defaults.

What's the advantage you see here versus just returning `apply_filters( 'altis.consent.always_allow_categories', [ 'functional', 'statistics-anonymous' ] )`?

Reply:
Returning the filter is definitely neater with setting the default straight away rather then my initial attempt.

Reply:
👍  just double checking in case you'd seen some other reason for the implementation you had. In general a filter will always pass some default value that can then be modified so it's rare there's a need to deviate from that pattern..